app/controllers/grade_entry_forms_controller.rb

```diff
@@ -102,6 +102,12 @@ def update_grade
   # For students
   def student_interface
     @grade_entry_form = GradeEntryForm.find(params[:id])
+    if @grade_entry_form.is_hidden
+      render 'shared/http_status', formats: [:html],
+             locals: { code: '404', message: HttpStatusHelper::ERROR_CODE['message']['404'] },
```

Comment:
Line is too long. [94/80]<br>Align the elements of a hash literal if they span more than one line.